issues.json

```diff
@@ -27,6 +27,24 @@
       "url": "http://www.oreilly.com/programming/free/files/from-future-import-python.pdf",
       "cover": "img/cover_picking_python_version_manifesto.gif"
     },
+    {
+      "title": "Violent Python: A Cookbook for Hackers",
+      "author": "Syngress",
+      "authorUrl": "https://twitter.com/syngress",
+      "level": "Beginner",
+      "info": "Violent Python shows you how to move from a theoretical understanding of offensive computing concepts to a practical implementation.",
+      "url": "https://github.com/reconSF/python/blob/master/Syngress.Violent.Python.a.Cookbook.for.Hackers.2013.pdf",
+      "cover": "img/cover_violent_python.jpg"
+    },
```

Comment:
Is it a free book?

Reply:
It is a free Ebook, even though it is commercialized for those who want the physical version.